load-test/docker-start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+./flood-agent --token "$FLOOD_TOKEN" --grid "$FLOOD_GRID"
```

Comment:
This file does not start docker?  Is it still required?

Reply:
good catch, will remove